_templates/component/new/default-entry.ejs.t

```diff
@@ -0,0 +1,11 @@
+---
+to: src/components/<%= h.inflection.dasherize(component_name) %>/index.jsx
+---
+
+const <%= h.changeCase.pascal(component_name) %> = ({ children }) => (
+	<div className="c-<%= h.inflection.dasherize(component_name) %>">
```

Comment:
updated to include the classname in there as well with the placeholder scss class created